crud/get.lua

```diff
@@ -65,26 +62,25 @@ function get.call(space_name, key, opts)
     end
 
     local bucket_id = vshard.router.bucket_id_strcrc32(key)
-
-    local replicaset, err = vshard.router.route(bucket_id)
-    if replicaset == nil then
-        return nil, GetError:new("Failed to get replicaset for bucket_id %s: %s", bucket_id, err.err)
-    end
-
-    local results, err = call.rw(GET_FUNC_NAME, {space_name, key}, {
-        replicasets = {replicaset},
-        timeout = opts.timeout,
-    })
+    local results, err = call.rw_single(
```

Comment:
Looks like `ro_single` (and `call.ro)`.

Reply:
It was `call.rw()` originally, so I didn't change it. If you think about it, there is some danger while executing this on the replica. The changes may not have propagated after you change them on the master.

Reply:
OK, but please add a comment with this problem description.